tools/parallel_tests.sh

```diff
@@ -30,46 +30,15 @@ mkdir -p ${output} ${logs}
 pytest --quiet --collect-only "$@" | grep :: > ${alltest}
 ${split_command} --number "l/${PARALLELISM}" --numeric-suffixes ${alltest} ${session_prefixes}
 
-if (type tmux &> /dev/null); then
```

Comment:
😢

Reply:
oh, sorry, that was committed by accident.